doc/api/http.md

```diff
@@ -3187,6 +3187,74 @@ Removes a header that is queued for implicit sending.
 outgoingMessage.removeHeader('Content-Encoding');
 ```
 
+### `outgoingMessage.writeFile(file[, options], callback)`
+
+<!-- YAML
```

Comment:
Can we make this `Active Development` due to the performance implications?

Reply:
What is that?

Reply:
I was referencing stability index: https://nodejs.org/dist/latest-v21.x/docs/api/documentation.html#stability-index